Dockerfile

```
@@ -1,12 +1,29 @@
-FROM golang:1.18 as workspace
-WORKDIR /go/src/github.com/testcontainers/moby-ryuk
+# -----------
+# Build Image
+# -----------
+FROM golang:1.18-alpine3.16 as build
+
+# Go build env
+ENV CGO_ENABLED=0
+
+WORKDIR /app
+
+# Install source deps
 COPY go.mod go.sum ./
 RUN go mod download
-COPY . ./
-RUN make build
 
-FROM alpine:3.13.6
+# Copy source & build
+COPY . .
+RUN go build -v -ldflags '-s' -o /bin/ryuk
+RUN chmod +x /bin/ryuk
+
+# -----------------
+# Distributed Image
+# -----------------
+FROM alpine:3.16
+
 RUN apk --no-cache add ca-certificates
-COPY --from=workspace /go/src/github.com/testcontainers/moby-ryuk/bin/moby-ryuk /app
-CMD ["/app"]
+
+COPY --from=build /bin/ryuk /bin/ryuk
+CMD /bin/ryuk
```

```suggestion
CMD ["/bin/ryuk"]
```

We can use the exec form, right?